src/Language/Haskell/Liquid/GHC/Play.hs

```diff
@@ -10,12 +10,98 @@ import Prelude hiding (error)
 import           Control.Arrow       ((***))
 import qualified Data.HashMap.Strict as M
 import qualified Data.List           as L
+import qualified Data.Maybe          as Mb 
 
-import Language.Haskell.Liquid.GHC.API as Ghc hiding (substTysWith, panic)
+import Language.Haskell.Liquid.GHC.API as Ghc hiding (substTysWith, panic,showPpr)
 import Language.Haskell.Liquid.GHC.Misc ()
 import Language.Haskell.Liquid.Types.Errors
 
 
+-------------------------------------------------------------------------------
+-- | Positivity Checker -------------------------------------------------------
+-------------------------------------------------------------------------------
+
+getNonPositivesTyCon :: [TyCon] -> [(TyCon, [DataCon])]
```

Comment:
How about documenting the output of this function? Perhaps documenting an example of an input and an output would be telling.